port/zos390/omrintrospect.h

```diff
@@ -60,8 +64,11 @@
 
 typedef __mcontext_t_ thread_context;
 
-
+#if defined(__clang__)
+#pragma pack(push, 1)
+#else
 #pragma pack(packed)
+#endif
```

Comment:
https://github.com/eclipse/omr/blob/master/doc/CodingStandard.md#comment-conditional-compilation-directives-if-ifdef-ifndef-else-elif-endif
```suggestion
#if defined(__clang__)
#pragma pack(push, 1)
#else /* defined(__clang__) */
#pragma pack(packed)
#endif /* defined(__clang__) */
```

Reply:
added comments, also this is supposed to be (__open_xl__) , tweaked accordingly.